### DISCUSSION TITLE
HTTPS instead of HTTP

### DISCUSSION THREAD

**Discussion by killerbeat12** [frangoteam/FUXA#1263] [General]
Hi! i saw some commented code for HTTPS for nearby future :)
As we are living in a secure internet nowadays and most web browsers start with https:// directly and also give warnings that HTTP is not save, could this be a good feature for your product and maybe implemented in next release?

**Comment by sgocken** [comment]
I would suggest you use an nginx reverse proxy in front of fuxa or any other web app.
https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/

I think this is the right way to do it. This allows app devs to not have to reimplement SSL and have the https layer handled by a web server that is designed to handle all the SSL and security things.

You can also look at traefik, but it is better used on a container system like docker where you have a lot of containers running different apps.
I use traefik on my docker swarm, put my wildcard SSL cert in traefik and I don't have to setup the cert for each container. Just put in a few config lines in the compose.yml for traefik to know what to do and it is good to go.
